bosh/opsfiles/production.yml

```diff
@@ -20,3 +20,10 @@
   - name: smtp-private
     static_ips: ((terraform_outputs.production_smtp_private_ip))
 
+- type: replace
+  path: /variables/name=postfix_ssl/options/common_name
+  value: ((terraform_outputs.production_smtp_private_ip))
+
+- type: replace
+  path: /variables/name=postfix_ssl/options/alternative_names
+  value: [ ((terraform_outputs.production_smtp_private_ip)), "smtp.fr.cloud.gov" ]
```

Comment:
Does the SAN for smtp.fr.cloud.gov get used?

Reply:
Also, does the private ip need to be in both the common name and alternative names?

Reply:
I have plans of someday using it instead of IP addresses, because they are icky, but not at the moment.

Reply:
The IP technically doesn't need to be in both places, but I've had problems with some clients being angry that it wasn't <cough>Java</cough> in the past, so I think it's not a bad idea.

Reply:
Can we drop the unused SAN until it's used?

Reply:
Golang TLS client requires a SAN last time I looked, so we can't get rid of it.

Reply:
I'm talking about the unused SAN--the DNS name that we're not using. Can we keep the IP address and drop the DNS name?

Reply:
Yes, though I think it would be better for us to use the hostname rather than the IP, so I would like to keep it in so that when I figure out the right way to create "smtp.fr.cloud.gov", we can just start using it without having to regen the certs and distribute them out to everybody.